lib/companies/jobs.ex

```diff
@@ -42,7 +43,12 @@ defmodule Companies.Jobs do
       ** (Ecto.NoResultsError)
 
   """
-  def get!(id), do: Repo.get!(Job, id)
+  def get!(id) do
+    (j in Job)
```

Comment:
Same as above, what if the company is soft-deleted. Its job should not be available.